travis/install_llvm.sh

```diff
@@ -0,0 +1,14 @@
+export LLVM_PATH=${HOME}/llvm/clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-ubuntu-14.04
```

Comment:
Nice! Also, I think that since this is a large PR we should wait for @rhornung67 before merging.